cli/jobs/basics/hello-interactive.yml

```diff
@@ -0,0 +1,12 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+command: echo "hello world" | sleep 3600
+environment:
+  image: azureml:AzureML-sklearn-1.0-ubuntu20.04-py38-cpu@latest
+compute: azureml:cpu-cluster
+services: 
+  my_vscode: 
+    job_service_type: VS_code 
+  my_jupyter_lab: 
+    job_service_type: jupyter_lab 
+  my_tensorboard:
+    job_service_type: tensorboard
```

Comment:
probably in comments add how to do SSH here.

Reply:
"tensor_board"

Reply:
fixed + added ssh in comment